src/commands/convert/hdf2asff.ts

```diff
@@ -51,15 +52,30 @@ export default class HDF2ASFF extends Command {
     }
 
     if (flags.upload) {
-      const profileInfoFinding: any = converted.pop()
-      const client = new SecurityHubClient({region: flags.region})
+      const profileInfoFinding = converted.pop()
+      convertedSlices = sliceIntoChunks(converted, 100)
```

Comment:
Why is this assignment being duplicated from the flags.output section?  Does not seem like that section modifies this object so reassignment here is unnecessary.  Would also allow us to maintain the const.